### DISCUSSION TITLE
Can Provider Reference use "Id" and include multiple Provider Group "Id" values?

### DISCUSSION THREAD

**Discussion by caseyj21** [CMSgov/price-transparency-guide#512] [Q&A] [answered]
The Provider Reference JSON schema does not show an "Id" as an option.
https://github.com/CMSgov/price-transparency-guide/blob/master/schemas/provider-reference/provider-reference.json

In the 3/16 Webinar and in other posts that I have reviewed, it appears that it is possible to add an "Id" value into the provider reference file, but we have had some issues with the validation tool when adding multiple groups and would like to confirm.

Is it possible to include multiple Provider Group "Id" values with many NPI/TIN values in each in a single Provider Reference file?  For example:
Provider Group "Id" 1
NPI 1, NPI 2, NPI 3
TIN 1

NPI 4, NPI 5, NPI 6
TIN 2

Provider Group "Id" 2
NPI 7, NPI 8
TIN 3

Provider Group "Id" 3
NPI 9
TIN 4

**Comment by shaselton-usds** [comment, marked as answer]
@caseyj21 No. Adding an ID would change the structure of the external provider reference group object/array. The external provider reference is an array of provider groups that are to be assigned to an ID within the in-network file.

Technically speaking, defining the `provider_group_id` within the in-network file's `provider_references` attribute is important as it is the "foreign key" that is referenced in `negotiated_rates` object. 

In your example, you will need to create three separate external provider reference files and give them "IDs" in the `provider_references` array within the "in-network" file.

**Reply by caseyj21** [reply]
Thank you.
